src/cascadia/WpfTerminalControl/WpfTerminalControl.csproj

```diff
@@ -23,6 +23,8 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.Shell.Framework" Version="16.7.30329.88" />
+    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="16.7.56" />

```

Comment:
This dependency reduces the portability of the WPF terminal control and, perhaps, makes it impossible(?) to run in the wpf terminal control test harness. Is it strictly necessary that we take a dep on VS libraries here?

Reply:
The only dependency on this package is the `ThreadHelper.Jtf.SwitchToMainThread()` in `ResizeAsync()` because I was getting "Object owned by different thread" exceptions. I tried using `Dispatcher.Invoke()` but that caused race conditions with resizing. 

Would you know of a better way to switch to the UI thread when the resize happens? I'd love to remove the dependency but couldn't find a way how.

Reply:
Just tested this and I'm still able to use the WPF test harness (WpfTerminalTestNetCore). As for portability, this dll should be portable as long as we include the shell framework package with the nuget package. We could do this while I investigate alternatives for switching to the UI thread in XAML.

Reply:
After fiddling a bit I found a way to use the dispatcher instead of Shell.Framework. This removes the dependency :)